plugin/EstateListHandle.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace onOffice\WPlugin;
+
+class EstateListHandle
+{
+
+	private $_listCustomField = array();
+	public function __construct()
+	{
+		$this->_listCustomField = [
+							'EN' => [
+										'objekttitel' => apply_filters('get_post_metadata','onoffice_title'),
+										'objektbeschreibung' => apply_filters('get_post_metadata','onoffice_description'),
+										'ort' => apply_filters('get_post_metadata','onoffice_city'),
+										'plz' => apply_filters('get_post_metadata','onoffice_postal_code'),
+										'objektart' => apply_filters('get_post_metadata','onoffice_property_class'),
+										'vermarktungsart' => apply_filters('get_post_metadata','onoffice_marketing_method'),
+										'Id' => apply_filters('get_post_metadata','onoffice_id')
+									],
+							'DEU' => [
+										'objekttitel' => apply_filters('get_post_metadata','onoffice_titel'),
+										'objektbeschreibung' => apply_filters('get_post_metadata','onoffice_beschreibung'),
+										'ort' => apply_filters('get_post_metadata','onoffice_ort'),
+										'plz' => apply_filters('get_post_metadata','onoffice_plz'),
+										'objektart' => apply_filters('get_post_metadata','onoffice_objektart'),
+										'vermarktungsart' => apply_filters('get_post_metadata','onoffice_vermarktungsart'),
+										'Id' => apply_filters('get_post_metadata','onoffice_datensatznr')
+									]
+						];
+	}
+	public function handleRecord($recordModified)
+	{
+		foreach ($recordModified as $key => $record) {
+			$lang = Language::getDefault();
```

Comment:
I think both English and German names always need to be accessible, otherwise things can break if two different users are using WP in different languages

Reply:
> This still needs to changed: Don't check which language is active. Just add all the fields of both languages.

@jmaas-onoffice field ‘objekttitel’ will contain value both custom field ‘onoffice_titel’ and ‘onoffice_title’. Link video demo : https://files.fm/f/yswfnrkg6. is this your mean ?

Reply:
Unfortunately it seems there was a misunderstanding. The video does not look like it is what we wanted.

I will prepare a description of what problem we want to solve and present this in the weekly with Linh.

Reply:
> Unfortunately it seems there was a misunderstanding. The video does not look like it is what we wanted.
> 
> I will prepare a description of what problem we want to solve and present this in the weekly with Linh.

i don't understand your mean, please demo detail your mean with video !